packages/database/index.node.ts

```diff
@@ -51,13 +51,19 @@ const ServerValue = Database.ServerValue;
  * @param app A valid FirebaseApp-like object
  * @param url A valid Firebase databaseURL
  * @param version custom version e.g. firebase-admin version
+ * @param nodeAdmin true if the SDK is being initialized from Firebase Admin.
  */
-export function initStandalone(app: FirebaseApp, url: string, version: string) {
+export function initStandalone(
+  app: FirebaseApp,
+  url: string,
+  version: string,
+  nodeAdmin: boolean = true
+) {
   /**
    * This should allow the firebase-admin package to provide a custom version
    * to the backend
    */
-  CONSTANTS.NODE_ADMIN = true;
+  CONSTANTS.NODE_ADMIN = nodeAdmin;
```

Comment:
We can remove this constant since RTDB is the only consumer of it.  If any other JS SDKs were used by the admin SDK in the future, they should not rely on this constant either.

Reply:
We should hold off on this. `firebase-admin` does not pin its dependency against `@firebase/database`, so we can only remove this in a major version bump to avoid breaking customers.

Reply:
Oh I think I misunderstood -- this is all internal in this package and utils

Reply:
@Feiyang1 so I still want `isNodeSdk()` to work ... where does `NODE_ADMIN` actually come from when it's not set in `initStandalone`?  Is it set in part of the build process?

Reply:
It defaults to `false` unless set by `initStandalone`. https://github.com/firebase/firebase-js-sdk/blob/master/packages/util/src/constants.ts#L30

Reply:
I'm not comfortable removing this at the moment ... it's used within `isNodeSdk()` which is used in many places including one place where `module.exports` is redefined.  I think removing this is likely to break something I don't understand.  Maybe someone from JSCore feels confident enough to do it as a follow up.

Reply:
The check for CONSTANTS.NODE_ADMIN is unnecessary in `isNodeSdk()` because `CONSTANTS.NODE_CLIENT` will be always true when `CONSTANTS.NODE_ADMIN` is true. Sure, I will do a follow up PR for this.